### ISSUE TITLE
testing on high-resolution image

### ISSUE THREAD

**Issue by GloriaZLQ** [MahmudulAlam/Automatic-Identification-and-Counting-of-Blood-Cells#5]
when  you use the model on high-resolution image and divide image into 3x3grid, how to process the cells at the edge of the image.

**Comment by MahmudulAlam** [comment]
@GloriaZLQ 
Hi. Model is used on the high-resolution image in [Prediction_on_HRI.py](https://github.com/MahmudulAlam/Automatic-Identification-and-Counting-of-Blood-Cells/blob/cd101e24e81e4a5ebb6ab74a29fc67fa30e3d9c1/Prediction_on_HRI.py#L18-L41) and partially visible cells at the edge are ignored by the model as in the training set edge cells are not annotated as ground truth cells.

**Comment by MahmudulAlam** [comment]
The ```Images\``` folder was missing high-resolution images previously. I have added them to the folder.

**Comment by GloriaZLQ** [comment]
I watched your result in README "Combined Output",found that one rbc cell was  counted twice and some rbc cells were ignored at the edge of the 3x3 grid.  I think this problem affects the counting accuracy and  do  you have any solution?

**Comment by MahmudulAlam** [comment]
That RBC cell is divided by grid line and two predictions are coming from two grid cells. Double counting can be prevented using proposed KNN and IOU based verification which is implemented only for platelets only. Those [high-resolution images (HRI)](http://www.biosigdata.com/?download=red-blood-cells-rbcs) are used only to verify the generalization of the learning and not used any similar image during training and HRI dataset only contains images, not any annotations. So, if you want to show any counting performance in those images, you have to manually annotate the cells first. 
Edge cells are ignored because in the training dataset edge cells are not considered as full cell and not annotated as ground truth cell.

**Comment by GloriaZLQ** [comment]
ok,thank you
